pkg/importer/data-processor_test.go

```diff
@@ -244,7 +244,8 @@ var _ = Describe("Data Processor", func() {
 		}
 		dp := NewDataProcessor(mdp, "dest", "dataDir", tmpDir, "1G", 0.055, false)
 		dp.availableSpace = int64(1500)
-		qemuOperations := NewFakeQEMUOperations(nil, nil, fakeInfoRet, nil, nil, resource.NewScaledQuantity(int64(1500), 0))
+		usableSpace := int64((1 - 0.055) * float64(dp.availableSpace))
```

Comment:
Would it be better to use the constant DefaultGlobalOverhead instead of hard code the `0.055` in all tests that are calculating the usable spaces? 

I think it is better to keep consistency. Also, I saw some discussion before about possibly increase the file system overhead parameter, for example, from 0.055 to 0.6.... So, it would be even easier to update it if needed.